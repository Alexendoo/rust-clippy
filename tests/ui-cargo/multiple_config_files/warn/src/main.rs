// check-pass
// normalize-stderr-test: "`.*(\.?clippy.toml)`" -> "`DIR/$1`"

fn main() {
    println!("Hello, world!");
}
