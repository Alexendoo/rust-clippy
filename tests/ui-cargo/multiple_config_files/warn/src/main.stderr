Using config file `DIR/clippy.toml`
Warning: `DIR/clippy.toml` will be ignored.
